ansible_events_ui/db/migrations/versions/20220816_3bc7f2aca642_add_roles_actions.py

```diff
@@ -0,0 +1,95 @@
+"""add_roles_actions.
+
+Revision ID: 3bc7f2aca642
+Revises: 2afab2d669fe
+Create Date: 2022-08-16 11:01:52.510359
+
+"""
+from alembic import op
+from ansible_events_ui.db.models import roles, actions, role_actions
+
+# revision identifiers, used by Alembic.
+revision = "3bc7f2aca642"
+down_revision = "2afab2d669fe"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+
+    op.bulk_insert(roles,
+                   [
+                       {'id': 1, 'name': 'Owner'},
```

Comment:
My main goal here is to have lookups in the `role_actions` table be very fast. How would you propose that we add predefined roles to this table?  Is a migration the right place?  I could put more code in the migration instead of hard coding ids.